helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixDataAccessor.java

```diff
@@ -427,8 +437,9 @@ public boolean removeProperty(PropertyKey key) {
 
             int bucketSize = property.getBucketSize();
             if (bucketSize > 0) {
-              // TODO: fix this if record.id != pathName
               String childPath = parentPath + "/" + record.getId();
+              validateBucketizedEnabled(childPath, true /*isRead*/);
```

Comment:
I guess we don't need a specific comment for a parameter like this in Java? It is kind of impacting the code readability, what do you think? If you like, I suggest you put it end of line or before line.
```
validateBucketizedEnabled(childPath, true); // true - is read
```

Reply:
I personally feel that when reading the code without compiler, having comment like this for boolean value is helpful.